schema/mysql/v57/temporal/schema.sql

```diff
@@ -97,6 +88,25 @@ CREATE TABLE task_queues (
   PRIMARY KEY (range_hash, task_queue_id)
 );
 
+CREATE TABLE transfer_tasks(
+  shard_id INT NOT NULL,
+  task_id BIGINT NOT NULL,
+  --
+  data MEDIUMBLOB NOT NULL,
+  data_encoding VARCHAR(16) NOT NULL,
+  PRIMARY KEY (shard_id, task_id)
+);
+
+CREATE TABLE timer_tasks (
+  shard_id INT NOT NULL,
+  visibility_timestamp DATETIME(6) NOT NULL,
+  task_id BIGINT NOT NULL,
+  --
+  data BLOB NOT NULL,
```

Comment:
do we want this to be MEDIUMBLOB for consistency, or are we okay with BLOB for timer tasks?